pycromanager/acquisitions.py

```diff
@@ -258,8 +259,12 @@ def __init__(
             startSequence. A common use case for this hook is when one want to send TTL triggers to the camera from an
             external timing device that synchronizes with other hardware. Accepts either one argument (the current
             acquisition event) or two arguments (current event, event_queue)
-        show_display : bool or str
-            If True, show the image viewer window. If False, show no viewer. If 'napari', show napari as the viewer
+        show_display : bool
+            If True, show the image viewer window. If False, show no viewer.
+        napari_viewer : napari.Viewer
+            Provide a napari viewer to display acquired data in napari (https://napari.org/) rather than the built-in
+            NDViewer. None by default. Data is added to the 'pycromanager preview' layer, which may be pre-configured by
```

Comment:
Maybe worth changing this away from "preview" to "data"? "view"? or something else?

Reply:
How about `pycromanager acquisition`?

Reply:
Sounds good!